src/styles/common.less

```diff
@@ -213,13 +216,13 @@ p {
 }
 
 .login-page {
```

Comment:
Color on the login page should be white. According to the EOS - black and colored backgrounds require white text.